alf/config_util.py

```diff
@@ -180,15 +188,16 @@ def is_used(self):
 _CONF_TREE = {}
 
 
-def config1(config_name, value, replacing_existing_config=False):
+def config1(config_name, value, mutable=True):
     """Set one configurable value.
 
     Args:
         config_name (str): name of the config
         value (any): value of the config
-        replacing_existing_config (bool): whether to replace with the provided
-            new value if the value has been configured before. A warning will be
-            generated when trying to set a value which has been set previously.
+        mutable (bool): whether the config can be changed later. If the user
+            tries to change an existing immutable config, the change will be
+            ignored and a warning will be generated. You can always change a
```

Comment:
Can add comments on the raised ValueError when trying to change immutable to mutable. Similar for ```config()```

Reply:
added.